deploy/kubernetes/elasticsearch/elasticsearch.gcloud.yaml

```diff
@@ -166,6 +166,61 @@ spec:
         resources:
           requests:
             storage: {{ ES_DATA_DISK_SIZE }}
+  - name: data
+    count:  {{ ES_DATA_OLD_NUM_PODS }}
```

Comment:
this is a direct copy-paste of the data configuration above, but changed out `ES_DATA_NUM_PODS` for `ES_DATA_OLD_NUM_PODS ` and `ES_DATA_DISK_SIZE` for `ES_DATA_OLD_DISK_SIZE`